charts/caddy-ingress-controller/templates/loadbalancer.yaml

```diff
@@ -6,10 +6,15 @@ kind: Service
 metadata:
   name: {{ include "caddy-ingress-controller.fullname" . }}
   namespace: {{ .Release.Namespace }}
+  {{- with .Values.loadBalancer.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
   labels:
     {{- include "caddy-ingress-controller.labels" . | nindent 4 }}
 spec:
   type: "LoadBalancer"
+  loadBalancerIP: {{ .Values.loadBalancer.loadBalancerIP }}
```

Comment:
I believe `loadBalancerIP` is [deprecated](https://github.com/kubernetes/kubernetes/pull/107235) since 1.24, so I think we should avoid using this. One common alternatives is to use annotation. Or does AWS only works with `loadBalancerIP`?

Reply:
No, AWS doesn't need it as far as I know. Any static/Elastic IPs are set with specific nlb annotations. I only added it to also solve issue #109. I thought K8s used it, but apparently not.

Reply:
Sorry about the delay on that. I added a comment on the deprecation of the spec to both the template and the values file so that it's obvious to anyone trying to use it after v1.23